frontends/open-discussions/src/Router.js

```diff
@@ -27,7 +29,9 @@ export default class Router extends React.Component<Props> {
         <Provider store={store}>
           <ReduxQueryProvider queriesSelector={getQueries}>
             <ReactRouter history={history}>
-              <ScrollToTop>{children}</ScrollToTop>
+              <ThemeProvider theme={combinedTheme}>
+                <ScrollToTop>{children}</ScrollToTop>
+              </ThemeProvider>
```

Comment:
When I moved CourseSearchbox to typescript, I got rid of the scss styling and used Styled Components instead. Some colors / media queries are part of a theme now, so we need to provide the theme.